be/src/olap/tablet.cpp

```diff
@@ -1752,6 +1761,7 @@ Status Tablet::prepare_compaction_and_calculate_permits(CompactionType compactio
         StorageEngine::instance()->create_base_compaction(tablet, _base_compaction);
         DorisMetrics::instance()->base_compaction_request_total->increment(1);
         Status res = _base_compaction->prepare_compact();
+        set_last_base_compaction_status(res.to_string());
```

Comment:
should also update status while `execute_compact` fail.

Reply:
done